winbuild/README.md

```diff
@@ -74,6 +74,8 @@ Open a Visual Studio Command prompt:
 where `<options>` is one or many of:
 
  - `VC=<num>`                    - VC version. 6 or later.
+ - `RTLIBCFG=<dll/static>`       - Runtime library configuration, defaults to dll

```

Comment:
RTLIBCFG we didn't document, I think to discourage its use since dll should be used most of the time for C runtime. We could document it but I think there should be another line with a warning.

Reply:
Thanks for the feekback. 
I have added a warning message for the RTLIBCFG flag.
Please check the image below, thanks.

<img width="834" alt="截圖 2021-08-18 下午3 47 29" src="https://user-images.githubusercontent.com/7500954/129859528-807cb6f9-a89f-4151-b65d-22869fbde890.png">